spec/models/miq_widget_spec.rb

```diff
@@ -177,6 +177,13 @@
           expect(result[@group1]).to match_array([@user1])
         end
 
+        it 'ignores a group that no longer exists' do
+          @user1.update(:miq_groups => [@group2])
```

Comment:
Is this so that when we delete group1 it doesn't fail due to being a non-empty group since user1 is in it?

Reply:
I removed this line by not using destroy since the idea was to leave a widget's subscription to be orphaned as the group was removed.  Previously, I had to workaround doing group1.destroy by making sure the existing user was moved from group 1 to 2.  By using delete, I don't need to do that.